src/azure-cli/azure/cli/command_modules/acr/replication.py

```diff
@@ -26,6 +29,7 @@ def acr_replication_create(cmd,
                            registry_name,
                            resource_group_name=None,
                            replication_name=None,
+                           region_endpoint_enabled=None,
```

Comment:
region_endpoint_enabled = True

Reply:
Would have done this but for some reason this causes the following help output. Not sure why the CLI renders the boolean True as a number.

![image](https://user-images.githubusercontent.com/15149427/83068550-6065c700-a01d-11ea-95ed-bf2a8f049171.png)

Reply:
Can you check how similar flags such as admin enabled is implemented?

Reply:
@djyou I thought I  responded, but just seeing I didn't Basically admin_enabled defaults to false in the signature (this isn't exposed automatically in CLI). public_network_enabled defaults to sdk default which is allowed/enabled. That being said we write the default in help text.

I suppose I can say default is true and leave to sdk to set default (sdk will set field to true if None is passed). I'll remove the defaulting logic I added, was paranoid.